### PR TITLE
Add vim-signify highlights

### PR DESCRIPTION
**Pull request by ranebrown**

### CODE REVIEW

File: colors/oceanicnext.vim [hunk 1]
```diff
@@ -407,6 +407,17 @@ hi! link GitGutterChangeDelete OcOrange
 
 " }}}
 
+" ----- Plugin (vim-signify) --------------------------------------------- {{{
```
Comment (adrian5) [inline colors/oceanicnext.vim]: Could you add two more dashes (`--`) to the line before `{{{` to align it with the other sections?

**Comment by adrian5** [comment]
Looks good to me @ranebrown, thanks!

If you could do the small fix I noticed, that would be great.

If you haven't done that before, the steps would be:

1. In your local repo make the change to `colors/oceanicnext.vim`
2. Run `git add colors/oceanicnext.vim`
3. Run `git commit --amend` and exit vim with `:wq` to write/confirm the change
4. Run `git push --force` to overwrite the current commit/PR

That should allow for a fresh PR that I can commit _as-is_.

File: colors/oceanicnext.vim (hunk 1 above)
Comment (ranebrown) [reply colors/oceanicnext.vim]: @adrian5 I made the requested change.

**Comment by adrian5** [comment]
Great, thanks Rane!

**Pull request merged**